pages/dr9/sga.rst

```diff
@@ -10,169 +10,757 @@
 
 .. contents::
 
-*Page author and contact:* `John Moustakas`_
+*Primary contacts:* `John Moustakas`_ and `Dustin Lang`_
 
 .. _`John Moustakas`: ../../contact/#other-experts
+.. _`Dustin Lang`: ../../contact/#other-experts
 
-The Siena Galaxy Atlas (SGA) is an angular diameter-limited sample of galaxies constructed as part of the DESI Legacy Imaging Surveys. It provides
-custom, wide-area, optical and infrared mosaics (in :math:`grz` and :math:`W1-W4`), azimuthally averaged surface brightness profiles, and both aperture and integrated
-photometry for a sample of approximately 400,000 galaxies over :math:`20{,}000\,\mathrm{deg}^2`.
+``Overview``
+============
 
-All the code used to generate and analyze the catalog is publicly available:
+The ``Siena Galaxy Atlas (SGA)`` is a multiwavelength atlas of approximately
+:math:`385,000` nearby galaxies selected based on their apparent angular
+diameter. 
+
+These galaxies are intrinsically large enough to be spatially resolved from our
+vantage point in the universe, providing a unique and powerful window into the
+fossil record of galaxy formation and evolution and the galaxy-halo
+connection. In addition, the atlas serves as a valuable angular map of the
+foreground extragalactic sky for cosmological surveys of distant galaxies like
+the `Dark Energy Spectroscopic Instrument (DESI)`_ survey.
+
+The ``SGA`` delivers precise coordinates, multiwavelength mosaics, azimuthally
+averaged optical surface brightness and color profiles, integrated and aperture
+photometry, model images and photometry, and additional metadata for the full
+sample. The primary sample relies on deep, wide-field :math:`grz` imaging from
+the `DESI Legacy Imaging Surveys DR9`_ as well as all-sky infrared imaging at
+:math:`3.4-22` microns from `unWISE`_.
+
+The 2020 version of the atlas, ``SGA-2020``, is primarily selected from the
+`Hyperleda extragalactic database`_ of *known* large angular-diameter galaxies,
+although future versions of the atlas will be supplemented with galaxies
+identified from the `DR9`_ imaging itself, which will mitigate the heterogeneity
+and surface brightness incompleteness of the parent sample.
+
+Coupled with existing and forthcoming optical spectroscopy from `DESI`_,
+particularly the Bright Galaxy Survey (BGS) of :math:`10` million galaxies
```

Comment:
This is definitely overkill on `:math:`.  It may even be slowing down rendering the page.